docs/content/direct/get-started.md

```diff
@@ -137,6 +133,30 @@ wec2_context=$wec2_name
 label_query_both=location-group=edge
 label_query_one=name=cluster1
 ```
+
+## Exercise KubeStellar
+
+Now that your system is running, you can try some example scenarios
+
+1. Define the needed shell variables, using either the settings output as the script completes or the settings shown just above from the step-by-step instructions). Their meanings are defined [at the start of the example scenarios document](example-scenarios.md#assumptions-and-variables).
```

Comment:
Minor typo here: there is an orphan closing parenthesis after "instructions"

Reply:
Thanks. Fixed.